tests/utils.py

```diff
@@ -144,6 +159,14 @@ def get_image_path(
             data = (data > 0.5).astype(np.uint8)
             if not data.sum() and force_binary_foreground:
                 data[..., 0] = 1
+        elif labels is not None:
+            data = (data * (len(labels) + 1)).astype(np.uint8)
+            new_data = np.zeros_like(data)
+            for i, label in enumerate(labels):
+                new_data[data == (i + 1)] = label
+                if not (new_data == label).sum():
+                    new_data[..., i] = label
```

Comment:
Are these lines necessary?

Reply:
Probably not. I saw on line 160 that if by chance the volume has no ones and `force_binary_background` is enabled then it will add a plane of ones. I thought the idea was to prevent a test from failing by random chance.

Although with the default shape of (10, 20, 30) giving 6000 voxels, the chance that any label will be missing is astronomically low. So this can probably be removed.

Reply:
Oh, I see. I was curious about setting the value of one full slice, but I guess it's not important. Let's leave it, as sometimes the default shape won't be used, so this line might be reached.

Reply:
Thats true. Although now that I think about it, it should probably be changed to `new_data.flatten()[i] = label` in a future PR. Otherwise it will go out of bounds if the last dimension of the volume is less than the number of desired labels.